### PR TITLE
16 navigation sidepanel

### PR DESCRIPTION
**Pull request by Cris2508**

### CODE REVIEW

File: src/components/Layout/Layout.jsx [hunk 1]
```diff
@@ -1,7 +1,35 @@
...
+
+  //Layout with Menu Icon
+  const [showMenu, setShowMenu] = useState(false);
 
+  const toggleMenu = () => {
+    setShowMenu(!showMenu);
+  };
+
+  return (
+    <>
```
Comment (DanForder) [inline src/components/Layout/Layout.jsx:19]: We don't need a fragment here anymore, as the div is the parent element

File: src/components/Layout/Layout.jsx [hunk 2]
```diff
@@ -1,7 +1,35 @@
...
+      <div className="layout">
+        {showMenu && (
+          <>
+            <div className="menu__shadow"></div>
```
Comment (DanForder) [inline src/components/Layout/Layout.jsx:23]: all the classes in this file should have a block of `layout`, so could this be changed to `layout__menu-shadow` ? or `layout__shadow`

File: src/components/Menu/Menu.jsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+import contact from "../../images/contact.svg";
+import discover from "../../images/discover.svg";
+import favourites from "../../images/favourites.svg";
+import home from "../../images/home.svg";
+import closeBtn from "../../images/close-btn.svg";
+import { Link } from "react-router-dom";
+
+const Menu = ({ toggleMenu }) => {
+  return (
+    <>
```
Comment (DanForder) [inline src/components/Menu/Menu.jsx:15]: don't need the fragment here as the div is one parent element

File: src/components/Menu/Menu.jsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      <div className="menu">
+        <div className="menu__header">
+          <img className="menu-appLogo" src={logo} alt="App Logo" />
```
Comment (DanForder) [inline src/components/Menu/Menu.jsx:18]: menu-appLogo to be lower-case-hyphenated and follow BEM:

`menu__app-logo`

File: src/components/Menu/Menu.jsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+          <h1 className="h1">LOCAL</h1>
+          <img
+            className="menu-close"
```
Comment (DanForder) [inline src/components/Menu/Menu.jsx:21]: follow BEM: `menu__close`

File: src/components/Menu/Menu.jsx [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+            src={closeBtn}
+            alt="Close Button"
+            onClick={toggleMenu}
+          />
+        </div>
+        <div className="menu-home">
```
Comment (DanForder) [inline src/components/Menu/Menu.jsx:27]: follow BEM: `menu__home`

File: src/components/Menu/Menu.jsx [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+          <Link className="menu__link" to="/home">
+            <div className="menu__page">
+              <img className="menu-logo" src={home} alt="Home Logo" />
```
Comment (DanForder) [inline src/components/Menu/Menu.jsx:30]: menu__logo

File: src/components/Menu/Menu.scss [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+    h1 {
+      font-family: $text-font-family;
+      font-size: $sub-header-text-size;
+      font-weight: $light-font;
+    }
+    h2 {
+      font-family: $text-font-family;
+      font-size: $default-text-size;
+      font-weight: $light-font;
+    }
```
Comment (DanForder) [inline src/components/Menu/Menu.scss:42]: When we're using BEM, no element selectors, everything should be a class

File: src/pages/Home/Home.jsx [hunk 9]
```diff
@@ -2,18 +2,19 @@ import Layout from "../../components/Layout/Layout";
...
-      <h2>Section Heading</h2>
-      <h3>Panel Heading</h3>
-      <p>
-        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Non error
-        dolor in nostrum minima odio a fuga saepe soluta adipisci perspiciatis
-        maxime tempora, temporibus beatae voluptas repellat rerum. Dignissimos,
-        necessitatibus.
-      </p>
+    <Layout isWithMenu={true}>
+      <div>
```
Comment (DanForder) [inline src/pages/Home/Home.jsx:6]: do we need this div? it doesn't have any styles associated with it

**Review by DanForder — CHANGES_REQUESTED** [review]
- Remove empty MenuIcon.scss file

File: src/components/Layout/Layout.jsx (hunk 1 above)
Comment (Cris2508) [reply src/components/Layout/Layout.jsx:19]: deleted

**Review by DanForder — APPROVED** [review]

**Review by GeorgeSwift16 — APPROVED** [review]

**Pull request merged**